lumicks/pylake/population/dwelltime.py

```diff
@@ -1328,7 +1360,16 @@ def jac_fun(params):
 
     # output parameters as [amplitudes, lifetimes], -log_likelihood
     current_params[fitted_param_mask] = result.x
-    return current_params, -result.fun
+
+    if use_jacobian:
+        std_errs = np.full(current_params.shape, np.nan)
+        std_errs[fitted_param_mask] = _calculate_std_errs(
+            jac_fun, constraints, num_free_amps, current_params, fitted_param_mask
+        )
+    else:
+        std_errs = np.full(current_params.shape, np.nan)
```

Comment:
I think you can get away with just this

```suggestion
    std_errs = np.full(current_params.shape, np.nan)
    if use_jacobian:
        std_errs[fitted_param_mask] = _calculate_std_errs(
            jac_fun, constraints, num_free_amps, current_params, fitted_param_mask
        )        
```